### DISCUSSION TITLE
legal HTTP header name

### DISCUSSION THREAD

**Discussion by Pierrelai99** [Budibase/budibase#6694] [Help]
Now I facing an issue when using rest-api to call, I used on postman can success call, but when used in budibase it show Query Error: agent is not a legal HTTP header name

I think this might be budibase error 

Pls get in help thanks

![image](https://user-images.githubusercontent.com/99549872/178921326-b3229c7d-d753-4a0a-a2e6-e9d71d762022.png)
![image](https://user-images.githubusercontent.com/99549872/178921356-3a834078-246c-45a0-be1f-6369166734cd.png)

**Comment by melohagan** [comment]
Hey @Pierrelai99 

We had a look but couldn't reproduce. What does the contents of your `Headers` tab look like? 

Have you upgraded Budibase and your app to the latest version?
